index.html

```diff
@@ -4446,27 +4447,27 @@ <h3>Verification</h3>
         <p>
 This section contains an algorithm that <a>conforming verifier
 implementations</a> MUST run when verifying a <a>verifiable credential</a> or a
-<a>verifiable presentation</a>. This algorithm takes a byte sequence (bytes
-<var>inputBytes</var>) and a media type (string <var>inputMediaType</var>) as
-inputs, and produces an object (<var>result</var>) that contains the
-following:
+<a>verifiable presentation</a>. This algorithm takes a sequence of bytes
+([=byte sequence=] <var>inputBytes</var>) and a media type
+([=string=] <var>inputMediaType</var>) as inputs, and produces a result
+([=map=] <var>result</var>) that contains the following:
```

Comment:
For wording, consider

```suggestion
([=string=] <var>inputMediaType</var>) as inputs, and returns a [=map=]
that contains the following:
```

Including `<var>result</var>` is fine, but readers don't usually need to know what variable the algorithm uses to store its return value.

This return type is also more like a [struct](https://infra.spec.whatwg.org/#structs) than a map, and the fact that it's repeated here and in Verify Securing Mechanism, and that I copied it to [w3c/vc-data-integrity#227](https://github.com/w3c/vc-data-integrity/pull/227/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R3050-R3066), makes me think it should be given a name and moved to https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1383.html#algorithms. Doing that in a separate PR is fine.

Note that, strictly, you refer to a struct's [items](https://infra.spec.whatwg.org/#struct-item) with `|myStruct|'s [=struct type/item name=]`, and you refer to map [entry](https://infra.spec.whatwg.org/#map-entry) [values](https://infra.spec.whatwg.org/#map-value) with <code>|myMap|["`key`"]</code>, but I think it's fine to use `.` notation with both: I can't imagine implementations diverging based on that.

Reply:
Acknowledged on the above, using your rewording. I expect these algorithms will continue to need to be cleaned up (editorially) during CR. One of those items will need to be consistency among all of the algorithms here (and in other specifications).